staging/src/k8s.io/apiserver/pkg/endpoints/filters/trace.go

```diff
@@ -50,3 +51,9 @@ func WithTrace(handler http.Handler, longRunningCheck genericapirequest.LongRunn
 		handler.ServeHTTP(w, req)
 	})
 }
+
+func traceFilterStep(ctx context.Context, msg string, field ...utiltrace.Field) {
+	if trace, ok := genericapirequest.TraceFrom(ctx); ok {
+		trace.Step(msg, field...)
```

Comment:
we should have added nil receiver handling to `Step(...)`.

Reply:
yeah, missed that